packages/evalite/src/index.ts

```diff
@@ -89,11 +89,27 @@ const runTask = async <TInput, TExpected>(
 export const evalite = <TInput, TExpected = TInput>(
   evalName: string,
   opts: Evalite.RunnerOpts<TInput, TExpected>
-) => {
+) => registerEvalite(describe, evalName, opts);
+
+evalite.only = <TInput, TExpected = TInput>(
+  evalName: string,
+  opts: Evalite.RunnerOpts<TInput, TExpected>
+) => registerEvalite(describe.only, evalName, opts);
+
+evalite.skip = <TInput, TExpected = TInput>(
+  evalName: string,
+  opts: Evalite.RunnerOpts<TInput, TExpected>
+) => registerEvalite(describe.skip, evalName, opts);
+
+function registerEvalite<TInput, TExpected = TInput>(
+  describeFn: typeof describe.only | typeof describe.skip | typeof describe,
+  evalName: string,
+  opts: Evalite.RunnerOpts<TInput, TExpected>
+) {
   // Eagerly run the promise before the dataset for
   // maximum concurrency
   const datasetPromise = opts.data();
```

Comment:
We need some mechanism for not running `opts.data()` if we're skipping the test.

This means we can probably only support `.skip`, not `.only` with this change. Since you can trivially check if the func has been skipped.